functions/Get-DbaDatabase.ps1

```diff
@@ -99,8 +99,7 @@ Returns databases on multiple instances piped into the function
         [parameter(ParameterSetName = "Encrypted")]
         [switch]$Encrypted,
         [parameter(ParameterSetName = "Status")]
-        [ValidateSet('EmergencyMode', 'Normal', 'Offline', 'Recovering', 'Restoring', 'Standby', 'Suspect')]
-        [string]$Status,
```

Comment:
It doesn't like that :( :
![image](https://cloud.githubusercontent.com/assets/5613670/26058507/14649720-3976-11e7-8340-7b67819d4b4e.png)

advice appears to be to user DynamicParameters, which are on the way out?

or enums? Are you happy they'd work far enough back?